trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/GravitinoErrorCode.java

```diff
@@ -36,6 +36,7 @@ public enum GravitinoErrorCode implements ErrorCodeSupplier {
   GRAVITINO_OPERATION_FAILED(22, EXTERNAL),
   ;
 
+  @SuppressWarnings("ImmutableEnumChecker")
```

Comment:
`ImmutableEnumChecker` will check whether a `Eunm` is immutable. 
Such cases will be recognized as an immutable object:
- All fields are final.
- The types of all fields of the enum are deeply immutable.
- The class annotated by `@com.google.errorprone.annotations.Immutable`.

We ignore `io.trino.ErrorCode` because it is outside of our project. We can add a `@Immutable` on it.

Reply:
@coolderli can you please add this comment to the PR description?

Reply:
done